client/client-multi/client-c/CMakeLists.txt

```diff
@@ -261,3 +290,11 @@ install(TARGETS kaac DESTINATION lib)
 add_subdirectory(Modules/doxygen)
 
 add_subdirectory(Modules/cppcheck)
+
+externalproject_add(kaa_rsa_keygen SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/tools/kaa_encryption"
```

Comment:
Let's do it by execute_process(), as we did in [sample apps](https://github.com/kaaproject/sample-apps/blob/c73d96982f2baa4441d8bb2799b9826c0e913f87/datacollectiondemo/source/c/CMakeLists.txt#L25)